core/config/config.go

```diff
@@ -6,22 +6,24 @@ import (
 	"context"
 	"crypto/rand"
 	"encoding/hex"
-	"encoding/json"
 	"net/url"
 	"time"
 
+	"github.com/golang/protobuf/proto"
+
 	"chain/core/rpc"
 	"chain/core/txdb"
 	"chain/crypto/ed25519"
 	"chain/database/pg"
-	"chain/database/sql"
-	chainjson "chain/encoding/json"
+	"chain/database/raft"
 	"chain/errors"
 	"chain/protocol"
 	"chain/protocol/bc"
 	"chain/protocol/state"
 )
 
+//go:generate protoc -I. -I$CHAIN/.. --go_out=. config.proto
```

Comment:
Would it be better to put this line at the top of the file? And/or to add a comment saying that this produces the code for the `Config` and `BlockSigner` types?